AmericanMedicalAssociation/ama-style-guide-2
# [EWL-4513] SG2 | Remove Duplicate Promo Patterns

## Ticket(s)

**Jira Ticket**
- [EWL-4513: SG2 | Remove Duplicate Promo Patterns](https://issues.ama-assn.org/browse/EWL-4513)

## Description
Removes the following from the SG2 codebase:

- **"Promo Email Subscription"** (was de-scoped in SG1 and should not have been migrated)
- **"Image Without CTA Description"** (is duplicative to existing variants)
- **"Promo Logo with White Background"** (is duplicative to existing variants)


## To Test
- `gulp serve`
- [ ] When I click "molecules" in the SG2 menu
--- Then I should NOT see "Promo Email Subscription"
--- And I should NOT see "Image Without CTA Description"
--- And I should NOT see "Promo Logo with White Background"

## Visual Regressions
No noticeable regressions from running `backstop test`. There were no backstop scenarios matching these variants, so no changes needed to be made to the test suite.

## Relevant Screenshots/GIFs
N/A

## Remaining Tasks
N/A

## Additional Notes
This PR can be merged independently of the other Promo-related PRs and has no d8 implications.

---

[Guidelines for Contribution](CONTRIBUTING.md)
[SG2 Standards](ama-style-guide-2/docs/standards.md)
